### ISSUE TITLE
dependencies Arch

### ISSUE THREAD

**Issue by budRich** [olemartinorg/i3-alternating-layout#11]
Hi just thought i let you know that the install instructions for arch is not working. There is no longer a package for `python-i3-git` and the link in the description is broken.

I got it to work with `sudo pip install i3-py` .  While troubleshooting this i had a conversation with the maintainer of `i3ipc-python` in #i3@freenode, **acrisci**. according to acrsci `i3ipc-python` is a better alternative. And it is at least more convenient to install then using pip, so you might consider porting it to `i3ipc-python`.

Really nice script, spiraling layout should be available (and enabled) by default in i3 imo.

**Comment by olemartinorg** [comment]
Hi @budRich!

Sorry about the extremely late reply here, I forgot about this issue after first noticing it... I looked into the `i3ipc` library and how easy it would be to rewrite this to use `i3ipc`, but it seems it's not trivial (to be honest I haven't worked with Python in years, and I never actually used this script myself - I just wrote it because someone on reddit asked if it would be possible).

I'll update the readme so as not to lead Arch users astray, at least. Thanks for the heads up!
